### PR TITLE
cleanup build matrix

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: Tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php [hunk 1]
```diff
@@ -24,9 +25,8 @@ public function setUp()
 
     public function testFormTwigTemplate()
     {
-        $twigExtension = $this->getContainer()->get('twig');
-        $twigExtension->initRuntime();
```
Comment (dbu) [inline Tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php]: this method was deprecated and is no longer supported. as far as i can tell we don't need to call it, right?

File: Tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php [hunk 2]
```diff
@@ -24,9 +25,8 @@ public function setUp()
...
-        $renderer = $twigExtension->getExtension('form')->renderer;
+        /** @var TwigRendererInterface $renderer */
+        $renderer = $this->getContainer()->get('twig')->getExtension('form')->renderer;
```
Comment (dbu) [inline Tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php]: this property is no longer public in symfony 3.2 . any idea how we can achieve this in a better way?

Comment (wouterj) [inline Tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php]:
```php
$renderer = $this->getContainer()->get('twig')->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer');
```

**Comment by wouterj** [comment]
ping @dbu Please start your friday with this 😄

**Comment by ElectricMaxxx** [comment]
oh yea ... The friday starts in one and a half hour.


Am 05.01.2017 um 22:02 schrieb Wouter J:

**Comment by dbu** [comment]
@wouterj would you have an idea what is going wrong here? i don't get the php 5.5. lowest build error, and i get even less why symfony 3.0 and 3.1 fail with phpcr setup errors (when the php 5.5 build with the oldest symfony 2.8 fails much less than 3.0 + 3.1)

**Comment by wouterj** [comment]
The lowest dep builds seem to be caused by a BC break in Twig, which was fixed in 1.30 (see https://github.com/twigphp/Twig/pull/2311 ). This also caused trouble for Symfony 3.1 builds.

I don't know why Symfony 3.0 fails, but shall we simply remove this test job from the build? (just like we did for all other packages)

**Comment by wouterj** [comment]
Reverted dropping old Symfony support. This is because Symfony <3.2 doesn't define the form renderer as runtime. The bug you were getting when deciding to commit *"this is getting ridiculous, lets just conflict with old twig versions"* was actually a very simple bug: You forgot `$renderer =`

**Comment by wouterj** [comment]
Green! 🎉 

Removed the Symfony 3.0 test. As Symfony 3.0 isn't maintained anymore, and Twig 2.0-dev had some changes before 2.0-stable, Symfony 3.0 doesn't support Twig 2.0 anymore. However, it still claims it does, resulting in failing tests.

**Comment by dbu** [comment]
yay, thanks a lot!

**Pull request merged**